app/src/main/res/values/strings.xml

```diff
@@ -392,4 +391,8 @@
     <string name="item_duration_hour">%1$d hr</string>
     <string name="item_duration_minute">%1$d min</string>
 
+    <!-- placeholders -->
+    <string name="url_podcasts">empty</string>
```

Comment:
How about making these real empty strings and if they are `null` hide the fields? If a brand specifies them they are shown. This way we don't need a separate config.

Reply:
Do you mean ""-empty or nonexistent?

Reply:
Well, we need a key if there is no brand overwrite:
`<string name="url_podcasts" />`
Should work as default or?

Reply:
Because I am getting compile errors when having them nonexistent

Reply:
Alright!